main.nf

```diff
@@ -338,7 +391,9 @@ else{
 }
 
 gradients
-    .into{gradients_for_prelim_bet; gradients_for_eddy; gradients_for_topup;
+    .into{gradients_for_prelim_bet; gradients_for_eddy;
+          gradients_for_prepare_topup;
+          gradients_for_prepare_dwi_for_eddy;
           gradients_for_eddy_topup; gradients_for_test_eddy_topup}
 
 readout_encoding
```

Comment:
This is the closest I can put this modification, you'll have to do the replacing by hand. Replace lines 403 to 405 with :

```
dwi_for_prelim_bet
    .map{ [it[0] + it[2]] + it }
    .join(gradients_for_prelim_bet.map{ [it[0] + it[3], it[1], it[2]] })
    .map{ it[1..-1] }
    .set{dwi_gradient_for_prelim_bet}
```